guide/english/cplusplus/map/index.md

```diff
@@ -67,8 +67,8 @@ size of map is 1
 Inserting data with insert member function.
 
 ```c++
-myMap.insert(make_pair("earth", 1));
-myMap.insert(make_pair("moon", 2));
+myMap.insert(pair<char, int> ("earth", 1));
+myMap.insert(pair<char, int> ("moon", 2));
```

Comment:
This change is not necessary.   The original use of `make_pair` is valid syntax.